src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -103,6 +109,7 @@ protected override string GetInputTooltip(int index)
             return AstFactory.BuildAssignment(
                 GetAstIdentifierForOutputIndex(0),
                 AstFactory.BuildFunctionCall(
+                    pythonEngineClass,
```

Comment:
👍

Reply:
This turns out to be an easy win. VM magic! @aparajit-pratap @mjkkirschner  Thank you for pointing this out. Saves me the time to implement reflection.